### ISSUE TITLE
VTK dependency is very strict

### ISSUE THREAD

**Issue by RobPasMue** [pyansys/pyfluent-visualization#94]
I am currently working on the PyAnsys metarepo (https://github.com/pyansys/pyansys_landing/pull/35) for allowing the import of all public PyAnsys libraries. I am having conflicts between this library and pyansys/pydpf-post(core), due to VTK requirements. Apart from the fact that we are imposing the use of an old version on DPF (which we should change if possible - see https://github.com/pyansys/pydpf-post/issues/107) I was also wondering why pyfluent-visualization imposes the use of VTK 9.1.0 exclusively. It seems like the use of any other version of VTK causes incompatibility problems with this library.

Is there any reason for strictly requesting version 9.1.0 in your ``pyproject.toml`` file?

Pinging @dnwillia-work and @seanpearsonuk for discussion.

**Comment by dnwillia-work** [comment]
Yes, I'm not certain.  Besides Sean @ajain-work might also know.

**Comment by ajain-work** [comment]
@RobPasMue I was getting random crashes. Frequency of crashes increased after I downgraded version. So I kept version to 9.1.0.

11 17:47:03.118 ( 2.222s) [ ]vtkWin32OpenGLRenderWin:713 ERR| vtkWin32OpenGLRenderWindow (000002B3289DCA00): failed to get valid pixel format.

However nobody else has reported this issue. It could be specific to my machine. May be we can remove strict VTK dependency.

**Comment by RobPasMue** [comment]
I would suggest trying to loosen the requirement if possible. Setting a minimum VTK req is fine, but it should not be the latest version only. Again, if possible. I have opened a PR in https://github.com/pyansys/pyansys-landing (https://github.com/pyansys/pyansys-landing/pull/38) for when this is solved. As I mentioned, problem right now is ``DPF`` forcing to be lower than ``9.1.0``. If you could manage to work from an earlier version (for example, ``>=9.0.3``), that'd work.

**Comment by RobPasMue** [comment]
And again, the problem is not strictly with you guys. It's just that strictly forcing a requirement typically leads to conflicts. 

On the other hand, restricting dependencies on the upper range, that is ``<=, <`` (as DPF is doing), is also not a good practice, but we will have to solve that on their repo.

Lower range restrictions regarding dependencies, that is  ``>=, >``, should be set (if possible) at the minimum compatible version.

**Comment by RobPasMue** [comment]
When the linked issue is solved, a new release would be needed. It can be a patch release if you prefer, since the involved changes are not that big. Thanks for taking care of it @ajain-work!

**Comment by RobPasMue** [comment]
@dnwillia-work @ajain-work @mkundu1 @seanpearsonuk 

In order to solve the issue we have to create a release taking into account this change. Would you be in favor of doing a patch release?

**Comment by dnwillia-work** [comment]
Yes that makes sense to me.

**Comment by RobPasMue** [comment]
Great. If you want me to do it, I'll go ahead and do so. It's fine by me! Happy to help
